Update footer board info

lserranoruber opened the pull request:
Changed the names from CheckOUT members to fOUTje members and added board pic

casperboone commented on outsite.nl/assets/images/photos/boards/board23.jpg:
Could you please throw this pic in tinyjpg.com to make it a bit smaller? 😊

lserranoruber replied:
this is after passing it through tinyjpg like 5 times 🥲 I'll throw it in a few more times but I don't think it'll get much smaller than 500kb

casperboone reviewed (commented):
Looks good! Minor thingy

thomcsmits reviewed (commented):
It makes sense to met that vicevoorzitter would be below voorzitter in the list, do you agree?

Also note that in Dutch, it's more commonly vicevoorzitter (as 1 word) (though both are accepted)

lserranoruber commented:
@thomcsmits: good points, I changed around the order so it makes more sense, and fixed the vicevoorzitter as one word.
@casperboone: resized the image to 332kb
